sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -895,14 +895,136 @@ def test_non_euclidean_kneighbors():
     # Raise error when wrong parameters are supplied,
     X_nbrs = neighbors.NearestNeighbors(3, metric='manhattan')
     X_nbrs.fit(X)
-    assert_raises(ValueError, neighbors.kneighbors_graph, X_nbrs, 3,
+    assert_raises(ValueError, neighbors.kneighbors_graph, None, 3,
                   metric='euclidean')
     X_nbrs = neighbors.NearestNeighbors(radius=radius, metric='manhattan')
     X_nbrs.fit(X)
-    assert_raises(ValueError, neighbors.radius_neighbors_graph, X_nbrs,
+    assert_raises(ValueError, neighbors.radius_neighbors_graph, None,
                   radius, metric='euclidean')
 
 
+def test_kneighbors_graph_nearest_neighbors():
```

Comment:
I think you just mean `test_kneighbors_graph_method`.

But I suspect you should really have a test of the X=None case (i.e. `def_test_neighbors_no_query` or some such), and ensure that the tests of the X != case elsewhere are sufficient; then the tests of the graph methods should really just be checking equivalence between the graph and non-graph methods for various values of `X_{train,test}`.